@ndlib/gatsby-theme-marble/src/components/Shared/Card/index.js

```diff
@@ -46,7 +46,7 @@ const Card = ({
             </div>
             <ExternalLinkIcon target={target} />
             <figcaption sx={sx.figcaption(wide)}>
-              <Styled.h3 dangerouslySetInnerHTML={{ __html: label }} />
+              <h2 sx={sx.label} dangerouslySetInnerHTML={{ __html: label }} />
```

Comment:
We're looking for `Styled.h3` in the unit test, so we need to fix `test.js`. Once that's done, 👍